AppCenterData/AppCenterDataTests/MSDataTests.m

```diff
@@ -1896,6 +1896,8 @@ - (void)testReadsFromRemoteIfExpiredAndOnline {
       });
   MSDocumentWrapper *expectedDocumentWrapper = [MSDocumentUtils documentWrapperFromData:testCosmosDbResponse
                                                                            documentType:[MSDictionaryDocument class]
+                                                                              partition:@"user-123"
```

Comment:
You don't need a custom prefixed user partition it's only used for the user table in the storage. These calls are more likely to use `kMSDataUserDocumentsPartition`.

Reply:
I was wrong the returned metadata from the document wrapper should contain the user-account.